experimental/packages/opentelemetry-sdk-metrics-base/src/InstrumentDescriptor.ts

```diff
@@ -15,9 +15,20 @@
  */
 
 import { MetricOptions, ValueType } from '@opentelemetry/api-metrics-wip';
-import { InstrumentType } from './Instruments';
 import { View } from './view/View';
 
+/**
+ * Supported types of metric instruments.
+ * @see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument
```

Comment:
maybe better to point to a tag/commit instead main which moves day by day.
applies also to other places.